docs/tutorials/data_pipeline.md

```diff
@@ -15,7 +15,7 @@ defines how to process the annotations and a data pipeline defines all the steps
 A pipeline consists of a sequence of operations. Each operation takes a dict as input and also output a dict for the next transform.
 
 We present a classical pipeline in the following figure. The blue blocks are pipeline operations. With the pipeline going on, each operator can add new keys (marked as green) to the result dict or update the existing keys (marked as orange).
-![pipeline figure](../../resources/data_pipeline.png)
+![](../../resources/data_pipeline.png)
```

Comment:
I found that the annotation to the figure will force show in `readthedocs`. So I remove the annotation (with no side-effect to the original figure).